web/src/index.css

```
@@ -6,7 +6,7 @@
   @font-face {
     font-family: Inter;
     font-weight: 400 700;
-    src: url('/fonts/Inter-Variable.woff2') format('woff2-variations');
+    src: url('/fonts/var.woff2') format('woff2-variations');
```

```suggestion
    src: url('/fonts/Inter.var.woff2') format('woff2-variations');
```
This should be `Inter.var.woff2` right?

---

Yes, good spotting. Updated with a bad pun commit message :D